_data/cryptocurrencies.yml

```diff
@@ -176,6 +176,13 @@ websites:
       software: Yes
       doc: https://localbitcoins.blogspot.com.au/2013/05/using-two-factor-authentication-on.html
 
+    - name: SatoshiTango
+      url: https://satoshitango.com/
+      img: satoshitango.png
+      tfa: Yes
+      software: Yes
+      doc: https://github.com/2factorauth/twofactorauth/pull/2321#issuecomment-273971890
```

Comment:
I'm hesitant to add this as the doc link, considering we have a [notes](https://twofactorauth.org/notes/) section on the website already for sites without pre-made docs. Should we make a new note for this site?